docs/guides.md

```diff
@@ -484,6 +500,546 @@ and refresh rate prior to connecting to an app. See
+following command.
+
+```bash
+sudo apt update
+sudo apt install openssh-client
+```
+
+Next we need to generate the keys we will use to connect to our SSH session. This is as simple as running the
+following in a terminal:
+
+```bash
+ssh-keygen
+```
+
+and simply pressing enter through the default options. This will place a file called id_rsa and id_rsa.pub
+in the hidden folder ~/.ssh/. This is the default key used when this user initiates an SSH session.
+
+Next, we'll copy that public key to the authorized_users file in the ~/.ssh/ directory. These are the public keys
+allowed to access this machine over SSH, and will allow us to establish an SSH connection with this user
+to the localhost.
+
+```bash
+cat ~/.ssh/id_rsa.pub >> ~/.ssh/authorized_keys 
+```
+
+@tip{If you also want any other machines (e.g. a laptop or steamdeck) to connect to your machine remotely over ssh,
+be sure to generate a pubkey on that machine and append it to the authorized_keys file like we did above.)}
+###### SSH Server Modifications
+
+We'll want to make a few modification to the SSH server on the sunshine host, both for convenience and security.
+
+Modify /etc/ssh/sshd_config with the following changes
+
+@tabs{
+  @tab{nano | ```bash
+    sudo nano /etc/ssh/sshd_config
+    ```}
+  @tab{vim | ```bash
+    sudo vi /etc/ssh/sshd_config
+    ```}
+
+Find the line with `PasswordAuthentication` and make sure it is set to `no` (removed the `#` if present.
+Then find the line `PubkeyAuthentication` and make sure it is set to `yes` and remove the `#` from the beginning
+if present. When you're done you should have these two lines in your config somewhere.
+
+```
+PubkeyAuthentication yes
+PasswordAuthentication no
+```
+
+@tip{Using publickey encryption for SSH connections significantly increases your protection against brute force
+attacks, and protects you against a rogue machine pretending to be your SSH server and stealing your password.}
+
+The next step is optional, but if you do not plan on connecting to your computer remotely via ssh and only have
+installed SSH for the purposes of using Sunshine, it's a good idea to disable listening for remote SSH connections.
+Do this by changing the following lines near the top of your sshd config:
+
+```
+#ListenAddress 0.0.0.0  
+#ListenAddress ::
+```
+
+To the following:
+
+```
+ListenAddress 127.0.0.1
+ListenAddress ::1
+```
+
+This will only allow SSH connections coming from your computer itself.
+
+@tip{on some distributions, the maintainers have added some files in /etc/ssh/sshd_config.d/ which is pulled into
+your sshd_config. These modifications can conflict with what we've just done. If you have any files in
+/etc/ssh/sshd_config.d/, make sure they do not include any of the changes we've just made or you will experience
+problems.}
+
+###### Quick Test and Accept Host Authenticity.
+
+Now that that's over with, let's reboot the machine and try to connect! Accept any warnings about the unidentified
```

Comment:
```suggestion
Next, let's reboot the machine and try to connect! Accept any warnings about the unidentified
```

I think the double "that" is perfectly acceptable grammar wise, but might be confusing or distracting. https://english.stackexchange.com/questions/3418/how-do-you-handle-that-that-the-double-that-problem

Reply:
Absolutely. I try to eliminate it any chance I can, I think this guide is showing I need to practice my proofreading a little bit more.

Thank you, these are all excellent edits.

Reply:
No worries! I think most of the suggestions were the same couple of changes in many places. Capitalization of Sunshine/Moonlight (wherever it's not a CLI command), standardization of the WoL acronym, and the missing `}` at the end of the tab sections were the majority of the comments if I remember correctly.